### ISSUE TITLE
Home app reports Ecobee sensors as 'not responding'

### ISSUE THREAD

**Issue by apexad** [vojtamolda/homebridge-ecobee3-sensors#15]
iPhone 6s Plus
iOS 10.2
Homebridge running on an Ubuntu
exclude_thermostat: true

I have an Ecobee3 with just 1 additional sensor. The temperature and occupancy sensor show up fine, and seem to work as expected.

However, the Details section on the first page of the Home app always says '2 accessories not responding' even though it also reports occupancy and temperature of the room correctly.

I deleted the persist folder and restarted homebridge, re-entered the key in MY APPS and it does not change anything.

My thought is that this may because I have exclude_thermostat to true, or because the items have no user changeable status (which may be why ecobee removed them)?

**Comment by vojtamolda** [comment]
Hello @apexad,

thanks for opening the issue. I've noticed this problem too and it seems to be happening to other people as well. Good thing is that besides the non-resposive message everything seems to be working fine and actually responding.

Temporary fix is to exclude the problematic accessories from the status. Access the details of the accessory and then click _Status and Notifications_ and un-select the _Include in Status_.

![status](https://cloud.githubusercontent.com/assets/8467740/21935704/1d5fbe20-d974-11e6-9ecf-17ef48d31ea2.jpg)

However, doing this will exclude the temperature/occupancy reports from showing up at the _Home_ app main screen status, so this fix is not without side-effects.

There's also an error message when adding new accessories in the `Home` app and I think both problems are related and have something to do with the HomeKit identification mechanism. The non-responsiveness is probably caused by an incorrect implementation of the `identify(...)` function somewhere. There's a callback that needs to be executed but it never happens for some reason.... I need to dig a little bit down into the details of this, but it's certainly annoying and it needs to be fixed.

**Comment by apexad** [comment]
Thanks for responding. However, just so you know this plugin has actually become obsolete.

http://9to5mac.com/2017/01/12/ecobee-remote-sensors-homekit/

I have requested my thermostat get updated and all the sensors are showing up now. It seems that this plugin now only be needed for those with earlier non-HomeKit ecobee3 thermostats.

Anyway though, the don't include in status thing is a great fix, so I will close this issue.

**Comment by vojtamolda** [comment]
Cool! That's great news. This was plugin was meant as only a temprary fix anyway... 😃

**Comment by apexad** [comment]
Yeah, it's great! I definitely recommend that you go in and contact ecobee support to update your Thermostat.

By the way, thanks for your work on this plugin. It gave me access to my sensors about a week early and was indeed a great temporary fix. Don't stop working on it. As I said, it will probably still be useful to those with earlier non-HomeKit thermostats.

**Comment by vojtamolda** [comment]
I'll reopen this issue. I think the non-responsiveness should be fixed anyway.

Per discussion in #16, the new Ecobee update 9.7.969 doesn't expose measurements from the thermostat unit. It only reports the remote sensors to HomeKit, so this plugin still makes sense for some people.

**Comment by vojtamolda** [comment]
This is still an open and ongoing issue. I tried to find out what's happening but no luck today... 😞

**Comment by jeffreyjue1** [comment]
Recently, my HomeKit setup blew up when I removed some devices from my ISY-994i and forgot to restart Homebridge for HomeKit to get the latest from the hub.  However, I noticed in the process that the Ecobee plug-in shows "No Response" when only the AppleTV4 is the hub, but this issue goes away when I activate my iPad Air as a 2nd hub.  I did this while trying to debug why a temperature automation only works when the Eve or the other "Home" app (not the iOS Home app) is open.
